### PR TITLE
i#4755: Test previously missing EVEX vmovq variants.

### PR DESCRIPTION
**Pull request by khuey**
Add decoding tests for the f30f7e and 660fd6 vmovq variants.

Issue: #4755

### CODE REVIEW

**Comment by khuey** [comment]
Tests as discussed at https://github.com/DynamoRIO/dynamorio/pull/4756#issuecomment-786884168

File: suite/tests/api/ir_x86.c [hunk 1]
```diff
@@ -1278,6 +1278,38 @@ test_x64_inc(void *dc)
     instr = INSTR_CREATE_inc(dc, opnd_create_reg(DR_REG_EAX));
     test_instr_encode(dc, instr, 2);
 }
+
+static void
+test_x64_vmovq(void *dc)
+{
+    /* 62 61 fd 08 d6 0c 0a vmovq  %xmm25[8byte] -> (%rdx,%rcx)[8byte] */
+    buf[0] = 0x62;
```
Comment (derekbruening) [inline suite/tests/api/ir_x86.c:1286]: How about `const byte dbuf[] = { 0x62, 0x61, 0xfd, 0x08, 0xd6, 0x0c, 0x0a };`?  Seems easier to read and more compact.

File: suite/tests/api/ir_x86.c [hunk 2]
```diff
@@ -1278,6 +1278,38 @@ test_x64_inc(void *dc)
...
+    buf[2] = 0xfd;
+    buf[3] = 0x08;
+    buf[4] = 0xd6;
+    buf[5] = 0x0c;
+    buf[6] = 0x0a;
+
+#    if VERBOSE
+    disassemble_with_info(dc, buf, STDOUT, true, true);
+#    endif
+    ASSERT(decode_next_pc(dc, buf) == (byte *)&buf[7]);
```
Comment (derekbruening) [inline suite/tests/api/ir_x86.c:1297]: `disassembler_to_buffer` plus `strcmp` for a more complete test?  I'm looking at test_opmask_disas_avx512, etc. in this file.

File: suite/tests/api/ir_x86.c (hunk 1 above)
Comment (khuey) [reply suite/tests/api/ir_x86.c]: Done.

File: suite/tests/api/ir_x86.c (hunk 2 above)
Comment (khuey) [reply suite/tests/api/ir_x86.c]: Done.

**Review by derekbruening — APPROVED** [review]
Thanks for adding more tests!

**Comment by khuey** [comment]
Hrm, I'm trying to dust this off and commit it but I don't know how to get rid of this AArch64 Precommit failure.

**Pull request merged**